mdpow/fep.py

```diff
@@ -855,19 +865,20 @@ def contains_corrupted_xvgs(self):
         :attr:`Gsolv._corrupted` as dicts of dicts with the component as
         primary and the lambda as secondary key.
         """
-        from itertools import izip
+
```

Comment:
removing itertools import due to incompatibility.

Reply:
really, that doesn't work under 3? check six

Reply:
Replaced the izip import in latest commit